README.txt

```diff
@@ -36,6 +36,8 @@ For this plugin to work correctly, please mention the following items as page me
 3. 'amount' with a minimum of 1 rupee.
 
 == Changelog ==
+= 1.2.5 =
+* Bug fix for  payment pop up.
```

Comment:
@yashgit891 can we include woocommerce version in which bug is arising . and remove a white space after for

Reply:
@abdulwahidsharief done.